modules/materials-page/materials-page.permissions.js

```diff
@@ -0,0 +1,18 @@
+const { allow } = require('graphql-shield');
+
+const { hasRoles } = require('../../utils/rules');
+const {
+  roles: { ADMIN, SUPERADMIN },
+} = require('../../consts');
+
+const materialsPermissionsQuery = {
+  getAllMaterialsBlocks: allow,
```

Comment:
please, check mb allow is value by default and we don't need this permission here